package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-fetch",
-  "version": "1.6.1",
+  "version": "1.6.2",
```

Comment:
Fixed. I thought GitHub would hide the discussion now due to it being on an outdated diff, but I still see it.